eos/form-factors/analytic-b-to-psd-dkmmo2008-impl.hh

```diff
@@ -197,6 +199,7 @@ namespace eos
 
         // hadronic parameters
         UsedParameter MB;
+        UsedParameter fB; // Added by Carolina for cross checks
```

Comment:
Fix comment and usage?

Reply:
And we thought we had fixed all of them! Fixed locally.